interactions/client.py

```diff
@@ -102,7 +102,7 @@ def __init__(
     @property
     def guilds(self) -> List[Guild]:
         """Returns a list of guilds the bot is in."""
-        return [Guild(**_, _client=self._http) for _ in self._http.cache.self_guilds.view]
+        return [Guild(**_) for _ in self._http.cache.self_guilds.view]
```

Comment:
What are we ditching the `_client` declaration here for? Would it not be useful in terms of DX for a bot developer to be able to do something to a guild by easily reading off of this property?

```py
for guild in bot.guilds:
    if guild.name == "foo":
        await guild.method()
```

Reply:
it seems to be present already

I can't see any modification in cache, but ``'Guild' got multiple values for '_client'``

Reply:
```py
Traceback (most recent call last):
  File "C:\Users\\Desktop\PycharmProjects\library\main.py", line 12, in on_ready
    print(bot.guilds)
  File "C:\Users\\Desktop\PycharmProjects\library\interactions\client.py", line 105, in guilds
    return [Guild(**_, _client=self._http) for _ in self._http.cache.self_guilds.view]
  File "C:\Users\\Desktop\PycharmProjects\library\interactions\client.py", line 105, in <listcomp>
    return [Guild(**_, _client=self._http) for _ in self._http.cache.self_guilds.view]
TypeError: interactions.api.models.guild.Guild() got multiple values for keyword argument '_client'
```